backend/modules/vector_db/qdrant.py

```diff
@@ -18,7 +18,7 @@ class QdrantVectorDB(BaseVectorDB):
     def __init__(self, config: VectorDBConfig):
         if config.local is True:
             self.local = True
-            self.location = ":memory:"
+            self.location = "http://localhost:6333"
```

Comment:
this requires docker which will not always be the case, instead you can make this persistent by using disk

Reply:
https://python.langchain.com/docs/integrations/vectorstores/qdrant#on-disk-storage